.github/workflows/azure_ci_check.yml

```diff
@@ -0,0 +1,72 @@
+name: Azure CI
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize, edited ]
+  issue_comment:
+    types: [ created, edited, deleted ]
+
+jobs:
+  check-azure-ci-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
```

Comment:
does this do a git clone? would n't that be expensive for each time?

Reply:
Kind of, it only checks out the commits of the PR and takes 2 seconds.  And actually, this is not needed (I followed the examples which have this step) for checking comments.  So I removed it.